EnglishFilter/sections/specific.txt

```diff
@@ -235,6 +235,7 @@ xxxparodyhd.net,movgotv.com,123kmovies.net,123movies.*,bluemovies.co,f5movies.to
 icutlink.com##form[method="post"] > br
 ||safelinku.com/js/web-script.js
 ||goaloo.*/mn/*.gif
+reddit.com##article[id] > a[href^="https://alb.reddit.com/"] ~ *
```

Comment:
To Mobile. On desktop I get no hit.

Reply:
I was just confused

Reply:
@Sergey-Lyapin @BlazDT Dou you still see it? I cannot find.

Reply:
In mobile sometimes.